### PR TITLE
chore: set ADR-66 to Living and added an Abstract.

### PR DESCRIPTION
**Pull request by AjimenezDCL**

### CODE REVIEW

File: content/ADR-66-emotes-system-for-renderer.md [hunk 1]
```diff
@@ -30,7 +32,7 @@ Additionally due to memory limitations, the users can now equip up to ten emotes
 ## Glossary
 
 - _Emote_: An emote is a special kind of wearable holding animations to be played by the users. The animation itself is contained in a .gltf or .glb model.
-- _Plugin_: A feature implemented in a pattern followed by the renderer: [ADR](/adr/ADR-56).
+- _Plugin_: A feature implemented in a pattern followed by the renderer: [ADR](./ADR-56-plugin-system.md).
```
Comment (menduz) [inline content/ADR-66-emotes-system-for-renderer.md:35]: Please roll back this link, the rendered version doesn't interpret links to sources

File: content/ADR-66-emotes-system-for-renderer.md [hunk 2]
```diff
@@ -51,15 +53,15 @@ A plugin has been implemented to react to changes in `emotesOnUse`, download the
 
 Any animation already retrieved and ready will be collected in a _Dictionary_ in the _DataStore_, allowing completely decoupling with the plugin itself for every _Animation_ consumer.
 
-![EmotesDiagram](resources/ADR-66/emotes-diagram.png)
+![EmotesDiagram](../public/resources/ADR-66/emotes-diagram.png)
```
Comment (menduz) [inline content/ADR-66-emotes-system-for-renderer.md:56]: Same here

File: content/ADR-66-emotes-system-for-renderer.md [hunk 3]
```diff
@@ -51,15 +53,15 @@ A plugin has been implemented to react to changes in `emotesOnUse`, download the
...
 
 \*When unloading animations, the disposal of GLTF takes effect after the removal of the anim (so every system using the animation can react before it gets unloaded). For readability sakes in the diagram, it has been placed inversely.
 
 ### Avatar
 
 Emotes are tied to the avatar itself. The ones equipped by a user will be retrieved along the other wearables in the profile and they must be treated specifically as emotes. Also, the bodyshape affects the representation used in the GLTF downloaded and due to the limitations in the legacy animation flow, animations must be prewarmed at an specific point in the avatar loading process.
 
-The Avatar System is complex enough to be explained in its own [ADR-65](/adr/ADR-65).
+The Avatar System is complex enough to be explained in its own [ADR-65](./ADR-65-avatar-system-for-renderer.md).
```
Comment (menduz) [inline content/ADR-66-emotes-system-for-renderer.md:64]: And here

File: content/ADR-66-emotes-system-for-renderer.md [hunk 4]
```diff
@@ -89,4 +91,4 @@ Giving the spotlight to the data itself, instead of the systems, permit easy dec
 Downloading animations, controlling their lifecycle and disposing them is the heavy lifting of this system. There's a handful of valid approaches to this problem and few reasons to pick one over another; even a singleton (forbidden word) would solve it in a fairly clean way.
 
 Since the plugin system is already consolidated in the renderer architecture, there's no need to stray apart from it. You can read about the benefits of using it
-[here](/adr/ADR-56).
```
Comment (menduz) [inline content/ADR-66-emotes-system-for-renderer.md:92]: Lastly here

**Review by menduz — CHANGES_REQUESTED** [review]
Awesome! The links refs should be rolled back. Rely on the rendered version to preview links and image references

File: content/ADR-66-emotes-system-for-renderer.md (hunk 1 above)
Comment (AjimenezDCL) [reply content/ADR-66-emotes-system-for-renderer.md:35]: Oohh since I was having troubles in the editor I thought they got outdated. Reverting them!

Comment (AjimenezDCL) [reply content/ADR-66-emotes-system-for-renderer.md:35]: Done! fixed all of them

**Review by menduz — APPROVED** [review]

**Pull request merged**